src/database/config/config.js

```diff
@@ -0,0 +1,15 @@
+require('dotenv').config();
```

Comment:
You are doing great work, but should I comment line by line to show you all the es6 conventions that you are not following?

Reply:
These ES5 conventions are provided by default by sequelize